extension/gulpfile.babel.js

```diff
@@ -96,6 +98,7 @@ gulp.task('babel', () => {
     .pipe($.babel({
       presets: ['es2015']
     }))
+    .pipe(browserify())
```

Comment:
Basically I had to use `module.exports` and `requires` across the codebase to be able to share between the extension and node. So with that in mind, I added browserify because I couldn't get rollup to play nicely with `require`s. I would be more than happy to drop it if you can point me to getting rollup to play ball.

PS I hate modern web toolchains.